lamindb/__init__.py

```diff
@@ -40,16 +50,15 @@
    Transform
    Run
 
-Records, labels, features & schemas
```

Comment:
This was a conscious change 2 days ago! ☺️

Reply:
I also noticed that and it's a merge error. Not sure how I ended up there but I'll ensure that there are no changes.